src/components/AnalysisPage/index.js

```diff
@@ -64,16 +64,17 @@ function AnalysisPage({
             }))}
             generateHref={({ postName }, index) => {
               const {
-                country: { slug: countrySlug }
+                country: { slug: countrySlug },
+                language
               } = takwimu;
               const href =
                 index === 0
                   ? `/profiles/[geoIdOrCountrySlug]` // if politics
                   : `/profiles/[geoIdOrCountrySlug]/[analysisSlug]`;
               const as =
                 index === 0
-                  ? `/profiles/${countrySlug}` // if politics
-                  : `/profiles/${countrySlug}/${postName}`;
+                  ? `/profiles/${countrySlug}?lang=${language}` // if politics
```

Comment:
This means all these links will have `lang=xx` even if the original url didn't have one
![S](https://user-images.githubusercontent.com/1779590/76529280-2277ee80-6483-11ea-8959-bfa0d6fb2e71.png)

Would have been _cool_ if we only add this if `language` is not the **default** language